xml/operations-maintenance-controller-lifecyclemanager_recovery.xml

```diff
@@ -145,9 +145,9 @@ ff02::2         ip6-allrouters</screen>
    </step>
    <step>
     <para>
-     Start the freezer-scheduler:
+     Start the freezer scheduler:
     </para>
-<screen>sudo systemctl start freezer-scheduler</screen>
+<screen>sudo systemctl start openstack-freezer-scheduler.service</screen>
```

Comment:
I think the service name is  openstack-freezer-scheduler.  sudo service openstack-freezer-scheduler works. sudo service openstack-freezer.scheduler.service could have problem.

Reply:
done. thx!